### DISCUSSION TITLE
Oryx now supports locking layers

### DISCUSSION THREAD

**Discussion by kaczordon** [manna-harbour/miryoku#301] [Q&A] [answered]
Would this allow the voyager keyboard to work properly?

https://blog.zsa.io/layer-lock/

**Comment by manna-harbour** [comment, marked as answer]
Thanks for the update!

These issues

> Add Button layer
> Add Extra layer
> Add Tap layer
> Add layer lock
> Change mouse button order

at https://github.com/manna-harbour/miryoku/issues/90 are just due to those being more recent features added to Miryoku Babel based Miryoku implementations, and Miryoku Oryx being maintained manually, but if Oryx supports `DF` then layer lock could be added to Miryoku Oryx in the same way.

However, the new layer lock feature in QMK and Oryx might be used as an alternative implementation to lock the current layer. 

For voyager see https://github.com/manna-harbour/miryoku/discussions/241.
